cmd/clusterctl/client/config/providers_client.go

```diff
@@ -302,6 +308,13 @@ func (p *providersClient) defaults() []Provider {
 			url:          "https://github.com/kubernetes-sigs/cluster-api-provider-nested/releases/latest/control-plane-components.yaml",
 			providerType: clusterctlv1.ControlPlaneProviderType,
 		},
+
```

Comment:
```suggestion
```

Reply:
I saw we kept a newline in between the different provider types like in line 284.